tabs/led_strip.js

```diff
@@ -64,26 +64,26 @@ TABS.led_strip.initialize = function (callback, scrollPosition) {
         usedWireNumbers.sort(function(a,b){return a - b});
         return usedWireNumbers;
     }
-    
+
     function process_html() {
-        
+
         localize();
 
         // Build Grid
         var theHTML = [];
         var theHTMLlength = 0;
         for (var i = 0; i < 256; i++) {
             if (semver.lte(CONFIG.apiVersion, "1.19.0"))
-                theHTML[theHTMLlength++] = ('<div class="gPoint"><div class="indicators"><span class="north"></span><span class="south"></span><span class="west"></span><span class="east"></span><span class="up">U</span><span class="down">D</span></div><span class="wire"></span><span class="overlay-t"> </span><span class="overlay-s"> </span><span class="overlay-w"> </span><span class="overlay-i"> </span><span class="overlay-color"> </span></div>');
+                theHTML[theHTMLlength++] = ('<div class="gPoint"><div class="indicators"><span class="north"></span><span class="south"></span><span class="west"></span><span class="east"></span><span class="up">U</span><span class="down">D</span></div><span class="wire"></span><span class="overlay-t"> </span><span class="overlay-s"> </span><span class="overlay-w"> </span><span class="overlay-v"> </span><span class="overlay-i"> </span><span class="overlay-color"> </span></div>');
```

Comment:
This shouldn't be changed, since this change in the firmware is not in firmware that is apiVersion 1.19 or older.